fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -719,6 +721,45 @@ class PhysicsSystem extends WorldSystem {
         JOLT.destroy(this._joltBodyInterface)
         JOLT.destroy(this._joltInterface)
     }
+
+    /**
+     * Creates a ghost object and a distance constraint that connects it to the given body
+     * The ghost body is part of the LAYER_GHOST which doesn't interact with any other layer
+     * The caller is responsible for cleaning up the ghost body and the constraint
+     *
+     * @param id The id of the body to be attatched to and moved
+     * @returns The ghost body and the constraint
+     */
+
+    public CreateGodModeBody(id: Jolt.BodyID): [Jolt.Body, Jolt.Constraint] {
+        const body = this.GetBody(id)
+        const bodyPosition = body.GetPosition()
+        const ghostPosition = new THREE.Vector3(bodyPosition.GetX(), bodyPosition.GetY() + 0.05, bodyPosition.GetZ())
+        const ghostBody = this.CreateBox(new THREE.Vector3(0.05, 0.05, 0.05), undefined, ghostPosition, undefined)
+
+        const ghostBodyId = ghostBody.GetID()
+        this._joltBodyInterface.SetObjectLayer(ghostBodyId, LAYER_GHOST)
+        this._joltBodyInterface.AddBody(ghostBodyId, JOLT.EActivation_Activate)
+        this._bodies.push(ghostBodyId)
+
+        const constraintSettings = new JOLT.SwingTwistConstraintSettings()
```

Comment:
I did some more in depth reading on the SwingTwist constraint, and I'm not sure it is the correct joint we're looking for. It technically removes 3 DOF and I believe limits 1 or 2 more. Luckily, I found the Point constraint which actually only removes the 3 translational DOF